src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: "
+                + "the radius is: "
+                + radius
+                + " and the color is: "
+                + color);
```

Comment:
Use the 'name()' method to get the String representation of enum constants instead of directly calling 'color' which may not be properly overridden.